docs/patterns/single-new-eks-observability-accelerators/single-new-eks-opensource-observability-graviton.md

```diff
@@ -0,0 +1,331 @@
+# Singe New EKS Cluster Open Source Observability Accelerator
+
+## Architecture
```

Comment:
Talk a little bit about Graviton. Also reference CDK blueprints for list of supported addons for additional software.